README.md

```diff
@@ -104,6 +105,7 @@ This will instruct it to use the most recent 1.x version of ACCP.
 For more information, please see [VERSIONING.rst](https://github.com/corretto/amazon-corretto-crypto-provider/blob/develop/VERSIONING.rst).
 
 ```xml
+<!-- TODO: once ACCPv2.0.0 has been released, we need to update the version field below to reflect this. -->
```

Comment:
Let's make this a public comment. Something like:

> The below snippet will pull in all versions of ACCP prior to the 2.0.0 release. Once that it released we recommend that everyone switch to the `[2.0,3.0)` specifier instead.

Reply:
sure